lms/templates/courseware/progress.html

```diff
@@ -80,7 +80,10 @@ <h2 class="title">${_("We're working on it...")}</h2>
                     <p class="copy">${_("We're creating your certificate. You can keep working in your courses and a link to it will appear here and on your Dashboard when it is ready.")}</p>
                   </div>
                   <div class="msg-actions"></div>
-                  %elif not is_id_verified or is_unverified:
+                  %elif missing_required_verification or is_unverified:
```

Comment:
But won't that not fix this bug yet? like if I have never photo-verified and I am enrolled in the honor mode of a course, I won't be able to see the button. And is it now redundant to check for `is_unverified` if we're also now checking it as part of `missing_required_verification`?

I'm starting to get confused :/